src/ima-state-machine.js

```diff
@@ -267,7 +267,7 @@ function onAdCompleted(options: Object, adEvent: any): void {
  */
 function onAdsCompleted(options: Object, adEvent: any): void {
   this.logger.debug(options.transition.toUpperCase());
-  if (this.playOnMainVideoTag() && this._contentComplete) {
+  if (this.playOnMainVideoTag() && this._contentComplete && this.config.reloadSourceOnComplete) {
```

Comment:
```suggestion
  if (this.playOnMainVideoTag() && this._contentComplete && !this.player.config.playback. playAdsWithMSE) {
```

Reply:
@yairans the config is passed as plugin config called `reloadSourceOnComplete`